src/pages/LogView.tsx

```diff
@@ -8,9 +8,9 @@ interface LogViewProps {
 }
 
 const LogView: React.FC<LogViewProps> = ({ logType }) => {
-  const { hasLogs } = useLogContext();
+  const { hasIngestedLogs } = useLogContext();
 
-  return hasLogs ? (
+  return hasIngestedLogs ? (
```

Comment:
It may be nice to add a different view when the logs are empty to indicate that it's not just an error rendering the logs. There's an [empty state LeafyGreen component](https://www.mongodb.design/component/empty-state/example/), or you could do something simpler